ogticrd/covid-safe-paths
# Feat: changing way notify positive user

#### Description:

In this ticket I did three things:
 - Deleting the covidId array from the local storage, now we're saving everything in the users array
 - Showing the user nickname when they receive a message and if the do not have one it'll ask them for one
 - sending a true to haveBeenNotified field in the database to each user that has been notified
#### Linked issues:

Fix: [253](https://trello.com/c/y6gXrBFT/253-as-a-user-i-want-to-have-my-own-check-mark-if-i-have-been-notified-about-laboratories-results-2)
#### How to test:

- `git pull && git checkout Feat/253-changing-way-notify-positive-user`
- Head over to report screen and report yourself with a negative covid Dominican ID
- it'll should ask you for a nickname and then move you to the form
- change your state from negative to positive in the database and close and open the app
- it'll should show you a message telling you that you become a positive covid user
- if you are the owner of the cellphone it should ask you to share your location otherwise not 


***note***: You can use postman to set your id as positive
